.gitignore

```diff
@@ -1,2 +1,3 @@
 apps/slackbot/.env
-**/_build
\ No newline at end of file
+**/_build
+.env
```

Comment:
Since there is already apps/slackbot/.env, why don't we change it to `**/.env` to ignore these files in all subfolders?